prime-dotnet-webapi/Services/EmailService.cs

```diff
@@ -195,26 +205,32 @@ public async Task<IEnumerable<string>> GetPharmaNetProvisionerNamesAsync()
 
         private async Task Send(string from, string to, string subject, string body)
         {
-            await Send(from, new[] { to }, new string[0], subject, body, Enumerable.Empty<Attachment>());
+            await Send(from, new[] { to }, new string[0], subject, body, Enumerable.Empty<(string Filename, byte[] Content)>());
         }
 
-        private async Task Send(string from, string to, string subject, string body, IEnumerable<Attachment> attachments)
+        private async Task Send(string from, string to, string subject, string body, IEnumerable<(string Filename, byte[] Content)> attachments)
         {
             await Send(from, new[] { to }, new string[0], subject, body, attachments);
         }
 
-        private async Task Send(string from, IEnumerable<string> to, string subject, string body, IEnumerable<Attachment> attachments)
+        private async Task Send(string from, IEnumerable<string> to, string subject, string body, IEnumerable<(string Filename, byte[] Content)> attachments)
         {
             await Send(from, to, new string[0], subject, body, attachments);
         }
 
-        private async Task Send(string from, IEnumerable<string> to, IEnumerable<string> cc, string subject, string body, IEnumerable<Attachment> attachments)
+        private async Task Send(string from, IEnumerable<string> to, IEnumerable<string> cc, string subject, string body, IEnumerable<(string Filename, byte[] Content)> attachments)
         {
             if (!to.Any())
             {
                 throw new ArgumentException("Must specify at least one \"To\" email address.");
             }
 
+            // TODO temporarily only send emails to Anais just in case
+            var tempTo = new List<string>();
+            tempTo.Add("anais.hebert@nttdata.com");
+
+            to = tempTo;
```

Comment:
Testing code that needs to be removed.

Reply:
Just here temporarily to make sure no one accidently sends a real email to someone until testing is done